src/core_modules/capture-core/components/D2Form/D2Form.types.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import type { RenderFoundation } from '../../metaData';
```

Comment:
remember we had at some point this tool to complain about some imports that they were unused. This is fixing it 🎉 
I prepared a PR to fix _all_ of them 👍 
https://github.com/dhis2/capture-app/pull/873